developerportal/urls.py

```diff
@@ -16,7 +16,7 @@
     url(r"^admin/", include(wagtailadmin_urls)),
     url(r"^documents/", include(wagtaildocs_urls)),
     url(r"^article-feed/", RssFeeds()),
-    url(r"^oauth/", include("social_django.urls", namespace="social")),
+    url(r"^auth/", include("mozilla_django_oidc.urls")),
```

Comment:
Perhaps not _the best_ path naming convention, but at least it's not clashing with Django or Wagtail's own urls